src/audio/codec_adapter/codec/waves.c

```diff
@@ -330,8 +330,8 @@ static int waves_effect_init(struct comp_dev *dev)
 	waves_codec->o_format = waves_codec->i_format;
 
```

Comment:
Can you please mention why we need this change in the commit message?

Reply:
We found out that this component ignore the topology setting, this PR backports the fix to a stable Intel product release.